CFForGitHub.cs

```diff
@@ -0,0 +1,13 @@
+namespace CFDemo
+{
+    using System;
+
+    class CFForGitHub
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Does CF work with GitHub pull requests?");
+			Console.WriteLine("Hell yeah!");
```

Comment:
Why should we, sometimes it is okay to brag!

---
In reply to: [61097541](https://github.com/kivancmuslu/CFTest/pull/9#discussion_r61097541) [](ancestors = 61097541)